static/static.js

```diff
@@ -574,3 +574,6 @@ $("#bithub-events-embed").upcomingEvents({
   apiKey: "AIzaSyBsNpdGbkTsqn1BCSPQrjO9OaMySjK5Sns",
   calendarId: "jupiterjs.com_g27vck36nifbnqrgkctkoanqb4@group.calendar.google.com"
 });
+
+// Set year in footer copyright link
+$('.footer-copyright .end-year').text(new Date().getFullYear());
```

Comment:
Yes. I suppose there isn't much use in having JavaScript update it again on page load. At first I though maybe if it doesn't get built often it would be nice to have a fallback. But I guess that's not really a problem here. I will update the PR :)